torchx/schedulers/kubernetes_mcad_scheduler.py

```diff
@@ -0,0 +1,1121 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+
+This contains the TorchX Kubernetes scheduler which can be used to run TorchX
+components on a Kubernetes cluster.
+
+Prerequisites
+==============
+
+TorchX kubernetes scheduler depends on AppWrapper + MCAD and requires etcd intalled for distributed job execution.
+
+Install MCAD 
+
+.. Deploying Multi-Cluster-Application-Dispatcher
+
+    https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/main/doc/deploy/deployment.md
```

Comment:
just make this a link instead of a code block

Reply:
Updated in commit: 65362828973a37624a6774a6da36a7460b4f6a89